### ISSUE TITLE
Required field in forms

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#8086]
### Describe your problem(s)

![image](https://user-images.githubusercontent.com/20179540/82173126-db2a3600-98e9-11ea-8009-61eafc0209d5.png)

[WCAG21 1.4.1: Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html)
A form that uses color and text to indicate required fields

Assistive technology users should be able to identify the required fields easily.

### Describe your solution

Instructions at the top of the form explain how required fields and optional fields are displayed

"Required fields are marked with an asterisk ( * )" 
Red asterisk icon with "Required." text alternative, should follow label text
alt="Required"

"All the fields are required"

If most fields are required, indicate optional fields by displaying the word “optional” in parentheses next to the label text
"Optional fields are marked with (Optional) all the other fields are required"
